src/components/smart_objects/src/object_schema_item.cc

```diff
@@ -278,16 +302,22 @@ const SMember& CObjectSchemaItem::GetCorrectMember(
     const SMember& member, const utils::SemanticVersion& messageVersion) {
   // Check if member is the correct version
   if (member.CheckHistoryFieldVersion(messageVersion)) {
```

Comment:
Related to the comment above in `CheckCustomVehicleData`, but the main change here would adding the condition ` || (member.mSince != boost::none && member.mSince.get() > kModuleVersion)`

Reply:
@jacobkeeler maybe it is better to aggregate all checks regarding since/until in CheckCustomVehicleData.  Please take a look at 986bcf0

Reply:
Regarding 34fd6bd, this change does not make sense, now there is no check for the `since` value. Please revert this change.